lua/nvim-tree/notify.lua

```diff
@@ -36,9 +35,23 @@ do
   end
 end
 
+local createDefaultHandler = function()
```

Comment:
```suggestion
local create_handler = function()
```

No camel case please.